react/CozyTheme/styles.styl

```diff
@@ -0,0 +1,16 @@
+$transluentWhite=rgba(255, 255, 255, 0.88)
+
+.CozyTheme--inverted
+    background: var(--primaryColor)
+    --barIconColor white
```

Comment:
```suggestion
    --barIconColor var(--white)
```

Reply:
I am actually not sure this is a good idea to use CSS variables for base colors. This goes against the practice of using semantic colors in the theme. Otherwise as some point someone will override `white` to mean `black` or something to have the text on a button match what she wants, and then `white` would contain `black` and it would have no sense.

We should instead use a CSS variable referencing a contrast color. I've made the change to use `--primaryContrastTextColor`.

I had not make the change for buttons since we may use var(--white) to override the color in skins, but we sure should make the change in the future